pyro/util.py

```diff
@@ -380,6 +380,17 @@ def optional(context_manager, condition):
         yield
 
 
+class ExperimentalWarning(UserWarning):
+    pass
```

Comment:
@fritzo, @fehiepsi - I have introduced this warning for experimental interfaces, in this case the `predictive` function. Does this pattern seem okay?

Reply:
It is ok to me.